src/common/NoResults/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { Header, Wrapper } from "../Loader/styled";
+import { ReactComponent as Image } from "./NoResultsPicture.svg"
+
+export const NoResultsWrapper = styled(Wrapper)``;
```

Comment:
`export const NoResultsWrapper = styled(Wrapper)``; ` is ok but looks slightly wierd to me. If you want to use the same component but with defferent name you may also write in index.js import {Wrapper as NoResultsWrapper} from "../Loader/styled"; and it should be the same but it's not a big deal so up to you :)

Reply:
i did for eventual changes in the future

Reply:
I have the impression that you are complicating this whole component unnecessarily. It is enough, without any external imports.
```
import styled from "styled-components";
import { ReactComponent as Image } from "./NoResultsPicture.svg"

export const NoResultsWrapper = styled.div`
    max-width: 1368px;
    margin: 0 auto;
`;

export const NoResultsHeader = styled.h1`
    margin: 56px 0 0;
    font-size: 36px;
    margin-left: 20px
`;

export const NoResultsImage = styled(Image)`
    margin: auto auto; 
    display: block;

    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
        max-width: 100%;
        height: auto;
    };
`;
```